### ISSUE TITLE
[Feature]: Better documentation for what W&B does when logging a PyTorch tensor as a PIL Image

### ISSUE THREAD

**Issue by RylanSchaeffer** [wandb/wandb#7377]
### Description

This is W&B's documentation for Images: https://docs.wandb.ai/ref/python/data-types/image

There are two cryptic sentences: "Note : When logging a torch.Tensor as a wandb.Image, images are normalized. If you do not want to normalize your images, please convert your tensors to a PIL Image."

### Suggested Solution

Please say a little more about what this means. When you say images are normalized, what specifically is meant? Subtract the mean? Is the mean calculated per channel or over all three channels? Do we also divided by the standard deviations i.e. is this really standardization?

More documentation about this point would be appreciated!

**Comment by JoanaMarieL** [comment]
Hello @RylanSchaeffer , thank you for your suggestion we appreciate this. I already created a feature request and tagged our Docs team to check it. We'll provide an update for any progress about it.

**Comment by TonyPolich** [comment]
Agreed!  What is particularly unclear is if the images are normalised on a per image basis or per batch.

**Comment by mdlinville** [comment]
Checking comments for additional context
